ntfy/backends/slack.py

```diff
@@ -0,0 +1,9 @@
+from slacker import Slacker
+
+
+def notify(title, message, token, user, retcode=None):
+
+    slack = Slacker(token)
+
+    user_str = '@{}'.format(user)
```

Comment:
If I'm not mistaken, if you require the user to put the `@` in their config file, that also allows them to post to a `#achannel` instead.